### PR TITLE
Add Entropy plugin

### PR DESCRIPTION
**Pull request by shurl**
This PR adds the entropy plugin and updates the plugins version to 2.4.0, along with changelog updates for the previous two PR's.

### CODE REVIEW

File: debian/distros/bionic/control [hunk 1]
```diff
@@ -23,6 +23,19 @@ Description: The Server Density monitoring agent - ServerDensity CPU stats plugi
  .
  See https://www.serverdensity.com/ for more information.
 
+Package: sd-agent-entropy
+Architecture: all
+Breaks: sd-agent (<<2.2.0)
```
Comment (carlosperello) [inline debian/distros/bionic/control]: This is a new package, so no Breaks or Replaces entries are required.

File: debian/distros/jessie/control [hunk 2]
```diff
@@ -23,6 +23,18 @@ Description: The Server Density monitoring agent - ServerDensity CPU stats plugi
  .
  See https://www.serverdensity.com/ for more information.
 
+Package: sd-agent-entropy
+Architecture: all
+Breaks: sd-agent (<<2.2.0)
+Replaces: sd-agent (<<2.2.0)
```
Comment (carlosperello) [inline debian/distros/jessie/control]: ditto

File: debian/distros/stretch/control [hunk 3]
```diff
@@ -23,6 +23,18 @@ Description: The Server Density monitoring agent - ServerDensity CPU stats plugi
  .
  See https://www.serverdensity.com/ for more information.
 
+Package: sd-agent-entropy
+Architecture: all
+Breaks: sd-agent (<<2.2.0)
+Replaces: sd-agent (<<2.2.0)
```
Comment (carlosperello) [inline debian/distros/stretch/control]: ditto

File: debian/distros/trusty/control [hunk 4]
```diff
@@ -23,6 +23,18 @@ Description: The Server Density monitoring agent - ServerDensity CPU stats plugi
  .
  See https://www.serverdensity.com/ for more information.
 
+Package: sd-agent-entropy
+Architecture: all
+Breaks: sd-agent (<<2.2.0)
+Replaces: sd-agent (<<2.2.0)
```
Comment (carlosperello) [inline debian/distros/trusty/control]: ditto

File: debian/distros/xenial/control [hunk 5]
```diff
@@ -23,6 +23,18 @@ Description: The Server Density monitoring agent - ServerDensity CPU stats plugi
  .
  See https://www.serverdensity.com/ for more information.
 
+Package: sd-agent-entropy
+Architecture: all
+Breaks: sd-agent (<<2.2.0)
+Replaces: sd-agent (<<2.2.0)
```
Comment (carlosperello) [inline debian/distros/xenial/control]: ditto

File: entropy/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+Collect the amount of available entropy on a linux system.
+
+## Setup
+### Installation
+
+The entropy check can be installed with your package manager, if the sd-agent repository is configured on your server, [instructions are available on our support site](https://support.serverdensity.com/hc/en-us/search?query=entropy). To install the entropy check install the `sd-agent-entropy` package.
+
+### Configuration
+
+The entropy check does not require any configuration.
```
Comment (carlosperello) [inline entropy/README.md]: Permission setting qualifies as configuration, isn't it? This is confusing...

File: entropy/manifest.json [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{
+  "guid": "94588b23-111e-4ed2-a2af-14680eb587a3",
+  "linux_package_name": "sd-agent-entropy",
+  "maintainer": "hello@serverdensity.com",
+  "manifest_version": "0.0.1",
+  "max_agent_version": "3.0.0",
+  "min_agent_version": "2.1.1",
```
Comment (carlosperello) [inline entropy/manifest.json]: If packaging metadata states that the plugin package requires 2.2.0, this should follow it or the other version should be changed to use 2.1.1 instead.

Comment (shurl) [reply entropy/manifest.json]: Resolved

File: entropy/README.md (hunk 6 above)
Comment (shurl) [reply entropy/README.md]: I've updated the readme to clarify.

File: debian/distros/bionic/control (hunk 1 above)
Comment (shurl) [reply debian/distros/bionic/control]: Resolved.

File: debian/distros/stretch/control (hunk 3 above)
Comment (shurl) [reply debian/distros/stretch/control]: Resolved.

File: debian/distros/jessie/control (hunk 2 above)
Comment (shurl) [reply debian/distros/jessie/control]: Resolved.

File: debian/distros/trusty/control (hunk 4 above)
Comment (shurl) [reply debian/distros/trusty/control]: Resolved.

File: debian/distros/xenial/control (hunk 5 above)
Comment (shurl) [reply debian/distros/xenial/control]: Resolved.

File: packaging/el/inc/subpackages [hunk 8]
```diff
@@ -251,6 +251,21 @@ This package installs the elastic plugin.
 /usr/share/python/sd-agent/checks.d/elastic.py
 %config /etc/sd-agent/conf.d/elastic.yaml.example
 
+%package -n sd-agent-entropy
+Summary: Server Density Monitoring Agent. entropy plugin
+Group: System/Monitoring
+Requires: %{name} >= 2.2.0
```
Comment (carlosperello) [inline packaging/el/inc/subpackages]: Let's be consistent... .deb requires 2.2.2, .rpm should do it too...

Comment (shurl) [reply packaging/el/inc/subpackages]: Yup!

Fixed in latest commit.

**Pull request merged**